flexget/plugins/plugin_transmission.py

```diff
@@ -660,6 +662,7 @@ def on_task_exit(self, task, config):
         delete_files = bool(config['delete_files']) if 'delete_files' in config else False
         trans_checks = bool(config['transmission_seed_limits']) if 'transmission_seed_limits' in config else False
         tracker_re = re.compile(config['tracker'], re.IGNORECASE) if 'tracker' in config else None
+        directories_re = config['directories'] if 'directories' in config else None
```

Comment:
Can't you just use:
```python
directories_re = config.get('directories')
```
Also, do you need to update the schema to include this option?